gitops/charts/traffic-court-online/charts/maintenance-site/templates/deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "maintenance-site.fullname" . }}
+  labels:
+    app.openshift.io/runtime: nginx
+    {{- include "maintenance-site.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "maintenance-site.selectorLabels" . | nindent 6 }}
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      annotations:
+        checksum/keycloakConfig: {{ include "maintenance-site.tplvalues.render" ( dict "value" .Values.keycloakConfig "context" $ ) | sha256sum }}
+      labels:
+        {{- include "maintenance-site.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ .Values.global.licenseplate }}-vault
```

Comment:
I dont think we need to use the vault service account. This app will not use secrets if I understand correctly.

Reply:
Removed